crates/next-api/src/module_graph.rs

```diff
@@ -662,3 +701,58 @@ pub async fn get_reduced_graphs_for_endpoint(
     }
     Ok(result_vc)
 }
+
+#[turbo_tasks::function]
+pub async fn get_global_module_id_strategy(
+    project: Vc<Project>,
+) -> Result<Vc<GlobalModuleIdStrategy>> {
+    let graph_op = get_module_graph_for_project(project);
+    // TODO get rid of this once everything inside calls `take_collectibles()` when needed
+    let graph = graph_op.strongly_consistent().await?;
+    let _ = graph_op.take_collectibles::<Box<dyn Issue>>();
```

Comment:
You probably should wrap these three lines into a separate turbo-tasks function. `take_collectibles` would invalidate the whole function when the collectibles change, but we are actually not interested in that. You can use `resolve_strongly_consistent()` instead of `strongly_consistent` in that function and return that Vc